app/src/main/java/com/gerardojim/nuggetscalculator/ui/main/intent/MainIntent.kt

```diff
@@ -8,6 +8,4 @@ sealed class MainIntent {
         val hasDryFood: Boolean,
         val hasGreenie: Boolean,
     ) : MainIntent()
```

Comment:
MainIntent.kt is inheriting MainIntent()?

Reply:
@kelveenfabian 
```
sealed class MainIntent {
    data class Calculate(
        val selectedFoodPosition: Int,
        val dailyCaloricTarget: Int,
        val offset: Double = 0.0,
        val hasDryFood: Boolean,
        val hasGreenie: Boolean,
    ) : MainIntent()
 
 kelveenfabian yesterday 

MainIntent.kt is inheriting MainIntent()?

	Reply…
Resolve conversation
}